server/coordinator/procedure/create_partition_table.go

```diff
@@ -128,31 +132,39 @@ func (p *CreatePartitionTableProcedure) Start(ctx context.Context) error {
 			}
 			if err := p.fsm.Event(eventCreatePartitionTable, createPartitionTableRequest); err != nil {
 				p.updateStateWithLock(StateFailed)
-				return errors.WithMessage(err, "create partition table procedure create new shard view")
+				return errors.WithMessage(err, "create partition table")
 			}
 		case stateCreatePartitionTable:
 			if err := p.persist(ctx); err != nil {
 				return errors.WithMessage(err, "create partition table procedure persist")
 			}
 			if err := p.fsm.Event(eventCreateDataTables, createPartitionTableRequest); err != nil {
 				p.updateStateWithLock(StateFailed)
-				return errors.WithMessage(err, "create partition table procedure create partition table")
+				return errors.WithMessage(err, "create data tables")
```

Comment:
What is `data tables`?

Reply:
At present, the names of `subTable ` in CeresMeta and CeresDB are not uniform, which is named `dataTable` in CeresMeta, I will fix it and uniformly named `subTable`

Reply:
Ok.